modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -9787,20 +9788,17 @@ private void _testGetRelatedObjectEntriesWithRootObjectEntryId(
 
 		_addRoleUser(new String[] {ActionKeys.VIEW}, objectDefinitionA, _user);
 
-		Page<ObjectEntry> objectEntryPage = unsafeBiFunction.apply(
-			objectEntryA, objectRelationshipA_AA);
+		UnsafeSupplier<Page<ObjectEntry>, Exception> unsafeSupplier =
```

Comment:
Do we really need this unsafeSupplier?? I understand the reasons, but the porpose should not be this. Inline it.